timetracker/entries/models.py

```diff
@@ -1,25 +1,33 @@
 from django.db import models
-
 from django.utils import timezone
 
+class Client(models.Model):
+    name = models.CharField(max_length=200)
+
+    def __str__(self):
+        return self.name
 
 class Project(models.Model):
-    client = models.CharField(max_length=200)
+    client = models.ForeignKey('Client')
     name = models.CharField(max_length=200)
 
     def __str__(self):
         return '<{}> {}'.format(self.client, self.name)
 
-
-# Create your models here.
-class Entry(models.Model):
+class WorkPeriod(models.Model):
     start = models.DateTimeField(default=timezone.now)
     stop = models.DateTimeField(blank=True, null=True)
+
+    def __str__(self):
+        return '[{} - {}]'.format(self.start, self.stop)
+
+class Entry(models.Model):
     project = models.ForeignKey('Project')
+    workPeriod = models.ForeignKey('WorkPeriod', null=True)
```

Comment:
In Python we'd normally use `work_period` as the attribute name here. CamelCasing is normally reserved for class names.